tendrl/node_agent/alert/__init__.py

```diff
@@ -41,55 +41,60 @@ def update_alert(message):
+                    if not AlertUtils().equals(
+                        new_alert_obj,
+                        curr_alert
+                    ):
+                        existing_alert = True
+                        utils.update_alert_count(
+                            new_alert_obj,
+                            existing_alert
                         )
-                        utils.classify_alert(new_alert_obj, keep_alive)
-                        new_alert_obj.save(ttl=keep_alive)
-                    else:
-                        # Remove the clearing alert with same if exist
-                        utils.remove_alert(new_alert_obj)
-                        utils.classify_alert(new_alert_obj)
-                        new_alert_obj.save()
+                        if message.payload["alert_condition_unset"]:
+                            keep_alive = int(
+                                NS.config.data["alert_retention_time"]
+                            )
+                            utils.classify_alert(new_alert_obj, keep_alive)
+                            new_alert_obj.save(ttl=keep_alive)
+                        else:
+                            # Remove the clearing alert with same if exist
+                            utils.remove_alert(new_alert_obj)
+                            utils.classify_alert(new_alert_obj)
+                            new_alert_obj.save()
+                        return
                     return
-                return
-            # else add this new alert to etcd
-        if message.payload["alert_condition_state"] == \
-            constants.ALERT_SEVERITY["warning"]: 
-            utils.update_alert_count(
-                new_alert_obj,
-                existing_alert
-            )
-            utils.classify_alert(new_alert_obj)
-            new_alert_obj.save()
+                # else add this new alert to etcd
+            if message.payload["alert_condition_state"] == \
+                constants.ALERT_SEVERITY["warning"]:
+                utils.update_alert_count(
+                    new_alert_obj,
+                    existing_alert
+                )
+                utils.classify_alert(new_alert_obj)
+                new_alert_obj.save()
+            else:
+                logger.log(
+                    "error",
+                    NS.publisher_id,
+                    {
+                        "message": "New alert can't be a clearing alert %s" % (
+                            new_alert
+                        )
+                    }
+                )
         else:
-            logger.log(
-                "error",
-                NS.publisher_id,
-                {
-                    "message": "New alert can't be a clearing alert %s" % (
-                        new_alert
-                    )
-                }
-            )
+            # SDS native events
+            utils.save_utilization_only_alert(new_alert_obj)
```

Comment:
save utilization of what? This method name is confusing, simplify it and add helping comment

Reply:
`save_notification_only_alert`. My bad for creating confusion.